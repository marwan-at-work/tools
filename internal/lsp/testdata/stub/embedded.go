package stub

import (
	"context"
	"io"
)

var _ embeddedInterface = (*embeddedConcrete)(nil) //@suggestedfix("(", "quickfix")

type embeddedConcrete struct{}

type embeddedInterface interface {
	io.Reader
	context.Context
}
