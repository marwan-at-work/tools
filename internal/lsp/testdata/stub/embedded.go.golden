-- suggestedfix_embedded_8_27 --
package stub

import (
	"context"
	"io"
	"time"
)

var _ embeddedInterface = (*embeddedConcrete)(nil) //@suggestedfix("(", "quickfix")

type embeddedConcrete struct{}

// Deadline implements embeddedInterface
func (*embeddedConcrete) Deadline() (deadline time.Time, ok bool) {
	panic("unimplemented")
}

// Done implements embeddedInterface
func (*embeddedConcrete) Done() <-chan struct{} {
	panic("unimplemented")
}

// Err implements embeddedInterface
func (*embeddedConcrete) Err() error {
	panic("unimplemented")
}

// Value implements embeddedInterface
func (*embeddedConcrete) Value(key interface{}) interface{} {
	panic("unimplemented")
}

// Read implements embeddedInterface
func (*embeddedConcrete) Read(p []byte) (n int, err error) {
	panic("unimplemented")
}

type embeddedInterface interface {
	io.Reader
	context.Context
}

